src/bindings/interface/types.i

```diff
@@ -25,24 +25,24 @@ Only meant to be included, shouldn't be passed to cmake as a module!
 
 // Shared typemaps
 
```

Comment:
I'm guessing that the changes here are just adding new functions similar to the old ones, rather than actual changes??

Reply:
Yes, I have added one for `libcellml::Issue::Level `. As for the rest, I have cleaned things up and reordered things.

FWIW, for this PR, I started from PR #650 (since this is where I had introduced `INFORMATION`) and removed all the non-relevant bits. When it comes to `src/bindings/interface/types.i`, I can't recall the details, but I know that I couldn't get something to work unless I was using `$type` instead of say `libcellml::Units::Prefix`. Also, I reordered things as it makes it easier to maintain.